### ISSUE TITLE
UI stuff

### ISSUE THREAD

**Issue by bschmoker** [CoalitionBuilder/okcongress#5]
- show 'short name' by default in results (i.e. PATRIOT Act) - with "long name" in lowlighted description
- loading spinner for operations
- clickable 'committee name' to filter, instead of sidebar 
- oneclick X button to 'clear all filters'

**Comment by okdan** [comment]
1. An issue here, which may or may not be fixable in the short term, a not insignificant number of bills don't have a short name. At least not in the API results we're getting. So we defaulted to long name since it was easy. What we'd have to do is run a check on each one, and say IF there's a short name use that. ELSE show long name. But then you have hybrid results, and the sidebar filter language no longer makes sense :P 
2. Yes on spinner, and I made one that we could use. I think I pushed it into an /images/ directory on either the static_html branch or the frontend branch. or master...
3. This sounds good. Do we want it to replace the sidebar piece entirely?
4. Sounds good

**Comment by bschmoker** [comment]
1 - Ah, makes sense. Hmm, perhaps having an 'AKA' inline with the bill name, if a shortname exists?
 like "protect america by doing stuff" aka patriot act
2 - rad
3 - Hmm, suggest moving the "house v senate" and "dem v rep" to a bar above the results (with colors!)
 That would remove the need for a sidebar, IMO

**Comment by okdan** [comment]
I think it would be fine to show short name if we have it, or else show long name. I think that would get the job done for most people, and let us remove it from the filter list.

Not sure it would work to move the sidebar to the top, simply because we're going to be adding filters. Right now it's just two, and would work fine up there. But what happens when we add 3,4,5,6 more?

**Comment by bschmoker** [comment]
Good point on the number of filters... perhaps surfacing the 'most common' ones and having the 'advanced' ones show up when you click a another button?

![similar to github search, actually](http://goo.gl/4Ag4Bt)

**Comment by e01010100** [comment]
I'm all for making the short name the default but sometimes someone will have sponsored multiple bills with the exact same short name. Usually it has the year attached, but not always.
